.circleci/config.yml

```diff
@@ -84,23 +84,23 @@ commands:
 jobs:
   pretty-docs:
     executor:
-      class: small
+      class: medium
```

Comment:
CI would consistently silently bail on `yarn install`, this fixed that. We were likely running out of memory.

See "Install" step here silently stopping at "Fetch" step, causing the command to fail completely later.

https://app.circleci.com/pipelines/github/storybookjs/storybook/61775/workflows/4e8ba21c-8f6e-4d2b-b08d-46a6b0c764ab/jobs/588725